jobs/storage_test.groovy

```diff
@@ -22,12 +22,30 @@ job(name) {
   }
   parameters {
    stringParam('STORAGE_TYPE', 'gcs', "storage backend for helm chart, default is gcs")
+   stringParam('HELM_REMOTE_REPO', defaults.helm["remoteRepo"], "Helm remote repo name")
+   stringParam('HELM_REMOTE_BRANCH', defaults.helm["remoteBranch"], "Helm remote repo branch")
+   stringParam('HELM_REMOTE_NAME', defaults.helm["remoteName"], "Helm remote name")
+   stringParam('RELEASE', defaults.workflowRelease, "Release string for resolving workflow-[release](-e2e) charts")
   }
 
   triggers {
     cron('@daily')
   }
 
+  publishers {
```

Comment:
@smothiki noticed we weren't capturing logs; therefore this logic added (pulled from other test jobs)